hack/windows/Prepare-ServiceInterface.ps1

```diff
@@ -27,3 +28,10 @@ if (Get-NetAdapter -InterfaceAlias $INTERFACE_TO_ADD_SERVICE_IP -ErrorAction Sil
 $hnsSwitchName = $(Get-VMSwitch -SwitchType Internal).Name
 Add-VMNetworkAdapter -ManagementOS -Name $InterfaceAlias -SwitchName $hnsSwitchName
 Set-NetIPInterface -ifAlias $INTERFACE_TO_ADD_SERVICE_IP -Forwarding Enabled
+
+if ($StopKubeProxyOnCreation) {
+  # Restart kube-proxy to make new created inteface can be used.
+  # Kill kube-proxy and the process will be recreated by kube-proxy pod.
```

Comment:
Restart kube-proxy to ensure that the newly created interface can be used.
Kill kube-proxy and the process will be automatically restarted by the kube-proxy Pod.